server/config/app.ts

```diff
@@ -42,6 +55,32 @@ export default function createApp(): Express {
     app.use(express.static(path.join(__dirname, "../../client")));
     app.use(express.static(path.join(__dirname, "../../node_modules")));
 
+    //add support for cors
+    app.use(cors());
+
+    //setup express-session
+    app.use(session({
+        secret: ensuredEnv().SESSION_SECRET,
+        saveUninitialized:false,
+        resave: false
+    }));
+
+    // initialize flash
+    app.use(flash());
+
+    // initialize passport
+    app.use(passport.initialize());
+    app.use(passport.session());
+
+    // implement an Auth Strategy
+    passport.use(User.createStrategy());
+
+    // serialize and deserialize user data
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Nice work on this one. That's a false-positive type error. You might want to put a comment about it, though. Saying something like `// false-positive type error` would be enough.